prisma/schema.prisma

```diff
@@ -815,6 +822,24 @@ model CollectionReport {
   @@id([reportId, collectionId])
 }
 
+model BountyReport {
+  bountyId Int
+  bounty   Bounty @relation(fields: [bountyId], references: [id], onDelete: Cascade)
+  reportId Int    @unique
+  report   Report @relation(fields: [reportId], references: [id], onDelete: Cascade)
+
+  @@id([reportId, bountyId])
+}
+
+model BountyEntryReport {
+  bountyEntryId Int
+  bountyEntry   BountyEntry @relation(fields: [bountyEntryId], references: [id], onDelete: Cascade)
+  reportId      Int         @unique
+  report        Report      @relation(fields: [reportId], references: [id], onDelete: Cascade)
+
+  @@id([reportId, bountyEntryId])
```

Comment:
If we ever are to query reports for bounty entries than we would want to have a reverse lookup index on [bountyEntryId]